packages/convai-widget-core/src/types/attributes.ts

```diff
@@ -26,18 +34,12 @@ const MainAttributeList = [
   "override-first-message",
   "override-language",
   "override-voice-id",
+  "mic-muting",
+  "transcript",
+  "text-input",
+  "text-contents",
 ] as const;
```

Comment:
Are these (or can be) attributes `data-` prefixed? If not, it might cause some HTML validators to mark them as invalid. If that's the case, some customer might complain that they are chasing perfect SEO scores or whatnot, and that this shows up for them.